src/templates/finops-hub/modules/scripts/Recommendations.json

```
@@ -0,0 +1,74 @@
+{
+    "queries": [
```

Thinking ahead for when we support pulling data from Log Analytics and other systems, should we add a root-level "source" or something to indicate that we should run this against ARG? We also need to indicate what table this will go into, to save room for non-recommendation queries.
```suggestion
    "source": "AzureResourceGraph",
    "dataType": "Recommendations",
    "queries": [
```

---

Solved